src/scripts/main.js

```diff
@@ -2,7 +2,30 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+for (const person of people) {
+  const row = document.createElement('tr');
```

Comment:
you can simplify with insertAdjacentHTML
`row.insertAdjacentHTML('beforeend', ...and create all td here)`
and after
`table.append(row)`